home/views.py

```diff
@@ -1,37 +1,77 @@
-from django.shortcuts import render
-import meetup.api
-import datetime
-import time
-from pytz import timezone
-import pytz
-
-# Create your views here.
+from django.shortcuts import render_to_response
 
 
 def home_page(request):
-	
-	client = meetup.api.Client('73c42797541a6c207a2a2b41262a66');
-
-	group_info = client.GetGroup({'urlname': 'Perth-Django-Users-Group'})
-	group_events = client.GetEvents({ 'group_id': group_info.id , 'status' :  'upcoming'   })
-
-	keys = ['group_id','event_id','event_name','event_address','event_description','event_yes_rsvp_count', 'event_datetime']
-	array_event = []
-
-	for event in group_events.results: 
-		perth_tz = pytz.timezone('Australia/Perth')
-		my_date_in_perth =  datetime.datetime.fromtimestamp(event['time']/1000.0, perth_tz ).strftime('%Y-%m-%d %H:%M:%S') 
-		array_event.append({'group_id': group_info.id, 'event_id': event['id'], 'event_name': event['name'], 'event_address': event['venue']['address_1'], 'event_description': event['description'], 'event_yes_rsvp_count' : event['yes_rsvp_count'], 'event_datetime': my_date_in_perth   })
-		
-	
-	context = {
-		"title":    "Upcoming Events",
-		"tabtitle": "",
-		"group_info":     group_info.__dict__.keys(),
-		"group_name":     group_info.name,
-		"group_id":     group_info.id,
-		"group_events": array_event
-
-	}
-
-	return render(request, "home-index.html", context)
\ No newline at end of file
+    return render_to_response(
+        'home.html',
+        {
+        },
+
+        # context_instance=RequestContext(request)
+    )
+
+
+def get_involved(request):
+    return render_to_response(
+        'getinvolved.html',
+        {
+        },
+
+        # context_instance=RequestContext(request)
+    )
+
+
+def ajax_meetups_tab(request, event_status):
+    """
+    Queries the meetup.com API to get all the events of the status specified
+
+    :param request:
+    :param event_status: upcoming, past, proposed, suggested, cancelled, draft
+    :return:
+    """
+
+    import meetup.api
+    import datetime
+    import pytz
+
+    client = meetup.api.Client('73c42797541a6c207a2a2b41262a66')
+
+    group_info = client.GetGroup({'urlname': 'Perth-Django-Users-Group'})
+    group_events = client.GetEvents({'group_id': group_info.id, 'status': event_status})
+
+    array_event = []
+
+    event_count = 0
```

Comment:
I would personally count the events here instead of incrementing in the for loop below. `event_count = len(group_events.results)`, which saves you from writing `event_count += 1`.